rasa/core/policies/memoization.py

```diff
@@ -25,7 +25,7 @@
 )
 from rasa.core.featurizers.tracker_featurizers import FEATURIZER_FILE
 from rasa.shared.exceptions import FileIOException
-from rasa.core.policies.policy import PolicyPrediction, PolicyGraphComponent
```

Comment:
I know its just a placeholder, but maybe it'd be cleaner to have something like
```
class PolicyPrediction1:
  # old implementation

class PolicyPrediction2:
  # new implementation

PolicyPrediction = PolicyPrediction2
```

Reply:
We've used "2" or "GraphComponent" suffix pretty consistently and added #TODOs where stuff needs to be removed (both suffixes will disappear when we refactor/rename in the end)

Reply:
@JEM-Mosig This would basically invalidate the git history for the migrated classes. By duplicating we preserve at least large parts of the history